docs/cluster-overview.md

```diff
@@ -104,7 +104,7 @@ The following table summarizes terms you'll see used to refer to cluster concept
       <td>Application jar</td>
       <td>
         A jar containing the user's Spark application. In some cases users will want to create
-        an "uber jar" containing their application along with its dependencies. The user's jar
+        an "user jar" containing their application along with its dependencies. The user's jar
```

Comment:
I think uber jar is correct? which means contains dependencies (fat jar)

Reply:
Agree with @HyukjinKwon . An uber jar, also known as a fat JAR or JAR with dependencies.

Reply:
Oh! Sorry, I'm a newbie to Spark.
Thanks for pointing it out.